Description: Remove privacy breach in documentation
Author: Sophie Brun <sophie@freexian.com>
Last-Update: 2018-07-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/docs/index.rst
+++ b/docs/index.rst
@@ -2,25 +2,6 @@
 PySide
 ======
 
-.. image:: https://pypip.in/wheel/PySide/badge.png
-   :target: https://pypi.python.org/pypi/PySide/
-   :alt: Wheel Status
-
-.. image:: https://pypip.in/download/PySide/badge.png
-   :target: https://pypi.python.org/pypi/PySide/
-   :alt: Downloads
-
-.. image:: https://pypip.in/version/PySide/badge.png
-   :target: https://pypi.python.org/pypi/PySide/
-   :alt: Latest Version
-
-.. image:: https://pypip.in/license/PySide/badge.png
-   :target: https://pypi.python.org/pypi/PySide/
-   :alt: License
-
-.. image:: https://readthedocs.org/projects/pip/badge/
-    :target: https://pyside.readthedocs.org
-
 |
 
 Introduction
